lib/builder.js

```diff
@@ -4,6 +4,7 @@ var apiCompat = require('./api_compat')
 var heimdall = require('heimdalljs');
 var SilentError = require('silent-error');
 var BroccoliBuildError = require('./broccoli-build-error');
+var BroccoliNodeInfo = require('broccoli-node-info');
```

Comment:
Can you do this instead:

```
var getNodeInfo = require(‘broccoli-node-info’).getNodeInfo;
```

Seems like it would make the call sites simpler...